pages/common/az-repos.md

```diff
@@ -0,0 +1,21 @@
+# az repos
+
+> Manage Azure DevOps repos.
+> Part of `azure-cli`.
+> More information: <https://learn.microsoft.com/it-it/cli/azure/repos?view=azure-cli-latest>.
```

Comment:
```suggestion
> More information: <https://learn.microsoft.com/en-us/cli/azure/repos?view=azure-cli-latest>.
```
We generally prefer non-localized versions of Microsoft URLs.

Reply:
🤦🏻 My bad! I missed it, thank you for it